postgrest.cabal

```diff
@@ -24,6 +24,7 @@ Flag CI
 executable postgrest
   main-is:             PostgREST/Main.hs
   default-extensions:  OverloadedStrings, ScopedTypeVariables, QuasiQuotes, LambdaCase
+  ghc-options:         -threaded -rtsopts -with-rtsopts=-N
```

Comment:
It's pretty easy to experiment and see which options ghc check is OK with. For instance when I add Wall, Werror and O2 I get this

```
$ stack exec -- cabal check
These warnings may cause trouble when distributing the package:
* 'ghc-options: -Wall -Werror' makes the package very easy to break with
future GHC versions because new GHC versions often add new warnings. Use just
'ghc-options: -Wall' instead.

* 'ghc-options: -O2' is rarely needed. Check that it is giving a real benefit
and not just imposing longer compile times on your users.
```

So when I had those things in the cabal file before I was doing something that is discouraged.